README.md

```diff
@@ -1133,22 +1133,22 @@ Enable/Disable the batching evaluation of prefix advertisements to all peers wit
 Enable/Disable enforces the neighbor autonomous system to be the first AS number listed in the AS path attribute for eBGP. Valid values are 'true', 'false', and 'default'. On NX-OS, this property is only supported in the global BGP context.
 
 ##### `event_history_cli`
-Enable/Disable/specify size of cli event history buffer. Valid values are 'true', 'false', 'size_small', 'size_medium', 'size_large', 'size_disable' and 'default'. Size can also be specified in bytes.
+Enable/Disable/specify size of cli event history buffer. Valid values are false', 'size_small', 'size_medium', 'size_large', 'size_disable'. Size can also be specified in bytes. Please Note: 'default' and 'true' values are deprecated in 1.8.0 versions and idempotency is not guranteed for this property due to platform differences.

```

Comment:
I would reword these notes a little bit.

`**Please Note:** Setting this value to 'default' or 'true' has been deprecated in module version 1.8.0.  This property is only used for BGP debugging purposes and idempotency is not guaranteed.`